tailwind.config.cjs

```diff
@@ -3,7 +3,13 @@ module.exports = {
   purge: ['./src/**/*.{html,js}'],
   mode: 'jit',
   theme: {
-    extend: {}
+    extend: {
+      colors: {
+        gray: {
+          tt: '#72674c'
```

Comment:
Nice, I knew it would be something like this.  Thanks for taking the time to look into it!

Reply:
The icons are the same as before, I just changed the color.